website/docs/r/policy_tier1_gateway_interface.html.markdown

```diff
@@ -50,6 +50,7 @@ The following arguments are supported:
 * `mtu` - (Optional) Maximum Transmission Unit for this interface.
 * `ipv6_ndra_profile_path` - (Optional) IPv6 NDRA profile to be associated with this interface.
 * `urpf_mode` - (Optional) Unicast Reverse Path Forwarding mode, one of `NONE`, `STRICT`. Default is `STRICT`. This attribute is supported with NSX 3.0.0 onwards.
+* `site_path` - (Optional for global manager only) Path of the site the Tier0 edge cluster belongs to
```

Comment:
I think this can sound misleading (can be read as required for local manager, optional for global manager)

Reply:
Makes sense. I think it is better to mark it as "Optional" here, and explain in the description that this config only applies to Global Manager. BTW do you think we should make site_path a required param for global manager? Otherwise the Site of the interface would be non-deterministic.

> I think this can sound misleading (can be read as required for local manager, optional for global manager)

Reply:
I think if its required in UI, we should make it same.